server/pulp/server/content/web/settings.py

```diff
@@ -1,5 +1,45 @@
+import ConfigParser
+import logging
 import os
 
+from pulp.server import logs as pulp_logs
+from pulp.server import config
+
+
+try:
+    log_level = config.config.get('server', 'log_level')
+    log_level = getattr(logging, log_level.upper())
+except (ConfigParser.NoOptionError, AttributeError):
+    # If the user didn't provide a log level, or if they provided an invalid one, let's use the
+    # default log level
+    log_level = pulp_logs.DEFAULT_LOG_LEVEL
+
+LOGGING = {
```

Comment:
This appears to be duplicating the work in `pulp.server.logs.start_logging`; does django provide special handling of this LOGGER attr that you're using to emulate `start_logging`'s behavior? If so, could a comment be added to indicate why start_logging is used, and, if needed, if `start_logging` should be used as a reference for configuring django logging?